content/blog/2019-10-08-remote-read-meets-streaming.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Remote Read Meets Streaming
+created_at: 2019-10-08
+kind: article
+author_name: Bartlomiej Plotka
+---
+
+The new Prometheus version 2.13.0 is available and as always, it includes many fixes and improvements ([CHANGELOG](https://github.com/prometheus/prometheus/blob/release-2.13/CHANGELOG.md)). 
+However, there is one feature that some projects and users were waiting for: [chunked, streamed version of remote read API](https://docs.google.com/document/d/1JqrU3NjM9HoGLSTPYOvR217f5HBKBiJTqikEB9UiJL0/edit#heading=h.3en2gbeew2sa).  
+
+In this article I would like to present a deep dive of what we changed in the remote protocol, why and how to effectively use it.
+
+## Remote APIs
+
+Prometheus since the 1.x version had the ability to interact directly with its storage using [remote API](https://prometheus.io/docs/prometheus/latest/storage/#remote-storage-integrations). 
```

Comment:
```suggestion
Since version 1.x, Prometheus has had the ability to interact directly with its storage using [remote API](https://prometheus.io/docs/prometheus/latest/storage/#remote-storage-integrations). 
```